recipes/openmpi/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "2.0.0" %}
+{% set major = ".".join(version.split(".")[:2]) %}
+
+package:
+    name: openmpi
+    version: {{ version }}
+
+source:
+    fn: openmpi-{{ version }}.tar.bz2
+    url: https://www.open-mpi.org/software/ompi/v{{ major }}/downloads/openmpi-{{ version }}.tar.bz2
+    sha1: a34f07a0e94af4ccd8080d74827befd61824c6c3
+
+build:
+    number: 0
+    skip: True  # [win]
+    binary_relocation: true
+    detect_binary_files_with_prefix: true
+
+requirements:
+    build:
+        - perl
+        - gcc  # [linux]
```

Comment:
The mpich recipe requires gcc on unix (incl osx), is there a particular reason why you don't here?